examples/screenshot.rs

```diff
@@ -0,0 +1,48 @@
+use image::{DynamicImage, ImageOutputFormat};
+use libremarkable::framebuffer::common::*;
+use libremarkable::framebuffer::core::*;
+use libremarkable::framebuffer::*;
+use libremarkable::image::RgbImage;
+use std::fs::OpenOptions;
+
+fn main() {
+    let fb = Framebuffer::from_path("/dev/fb0");
```

Comment:
```suggestion
    let fb = Framebuffer::from_path(crate::device::CURRENT_DEVICE.get_framebuffer_path());
```

Autoselect the framebuffer so `rm2fb-client` is not needed when executing the command.

Reply:
Good call - done.